lucene/test-framework/src/java/org/apache/lucene/index/BaseTermVectorsFormatTestCase.java

```diff
@@ -504,8 +504,8 @@ protected void assertEquals(RandomTokenStream tk, FieldType ft, Terms terms) thr
           }
           try {
```

Comment:
Since you're cleaning up I think it'd be better to use assertThrows() with lambda...

Reply:
So it turns out be be less clean, since two of the implementations throw IllegalStateException and two throw AssertionError.

Reply:
@jpountz  may be interested in why this is inconsistent between implementations?